README.md

```diff
@@ -1,201 +1 @@
 # Terraform Provider Artifactory
```

Comment:
You have the word artifactory here - and you still need a readme. 

In your readme, include a link to API Documentation

Reply:
Readme will be added later, for now, added a link to the API documentation